diff --git a/android/build.gradle b/android/build.gradle
index 41b43599a9af6ef1c170047d39db1f3baa6ca682..d566e565cf91136132ee6cde8a9ad0ff3c368c12 100644
--- a/android/build.gradle
+++ b/android/build.gradle
@@ -247,7 +247,7 @@ dependencies {
     extractLibs "com.microsoft.onnxruntime:onnxruntime-android:latest.integration@aar"
   }
 
-  if (VersionNumber.parse(REACT_NATIVE_VERSION) < VersionNumber.parse("0.71")) {
+  if (REACT_NATIVE_VERSION.startsWith('0.') && REACT_NATIVE_VERSION.tokenize('.')[1].toInteger() < 71) {
     extractLibs "com.facebook.fbjni:fbjni:+:headers"
     extractLibs "com.facebook.fbjni:fbjni:+"
   }
